Source/Core/Core/HW/DSPLLE/DSPLLETools.cpp

```diff
@@ -59,14 +59,14 @@ bool DumpDSPCode(const u8 *code_be, int size_in_bytes, u32 crc)
 }
 
 // TODO make this useful :p
-bool DumpCWCode(u32 _Address, u32 _Length)
+bool DumpCWCode(u32 _Address, u32 _length)
```

Comment:
I'd have to make sure there isn't any variable conflict before I do that. I think the underscore is mainly used for external variables. Offhand, I don't know of any overlaps, but I would want to be careful to look out for instances such as [WII_IPC_HLE_Device.h](https://github.com/Stevoisiak/dolphin/blob/VariableConsistency/Source/Core/Core/IPC_HLE/WII_IPC_HLE_Device.h#L44-L52)